### PR TITLE
REL: 3.3.1 Bugfix release

### PR DESCRIPTION
**Pull request by sebix**
blocked by #2522 and #2523

The release dates need to be entered in NEWS and CHANGELOG before merging

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,15 +9,9 @@
 3.3.1 (unreleased)
```
Comment (aaronkaplan) [inline CHANGELOG.md]: I am missing the list of new things which work or were changed (i.e. shadowserver config parser -> empty string -> by default select all reports.

**Review by aaronkaplan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sebix) [reply CHANGELOG.md]: This file contains exactly that list, 10 lines below.

**Comment by sebix** [comment]
Rebased on develop and now ready to be merged after approval

File: CHANGELOG.md (hunk 1 above)
Comment (aaronkaplan) [reply CHANGELOG.md]: wait, I thought that's known bugs. Let me double check. Maybe I read the diffs too fast?

Damn, I did :) okay, then we are fine.

**Review by aaronkaplan — APPROVED** [review]
all good. thanks! Let's go.

**Pull request merged**